Makefile

```diff
@@ -53,9 +56,15 @@ all: $(TARGET)
 
 rebuild:
 	cargo build --manifest-path libportability/Cargo.toml --features $(BACKEND)
+ifdef SYMLINK_PORTABILITY
+	ln -sf "libportability.so" "target/release/$(SYMLINK_PORTABILITY)"
```

Comment:
isn't the source in `target/release` to begin with?
also, I think we need global paths here (so, with `$(CURDIR)` prepended)

Reply:
`ln -sf "libportability.so" "target/release/$(SYMLINK_PORTABILITY)"` adds a file called `target/release/libvulkan.so.1` which points to the file `libportability.so`.

According to the 'ln' man page: "Symbolic links can hold arbitrary text; if later resolved, a relative link is interpreted in relation to its parent directory." So `<whatever>/target/release/libvulkan.so.1`will point at '<whatever>/target/release/libportability.so`.

Anyways, I've prepended `$(CURDIR)`

Reply:
hmm, interesting. If that so, let's not have `$(CURDIR)` there, and I'll check (after merging) if Dota is OK with it

Reply:
Removed in next version. (Not pushed yet)